README.md

```diff
@@ -31,6 +31,27 @@ cache works between processes)
 
 ## Usage
 
+Add it in your root build.gradle at the end of repositories:
```

Comment:
>at the end repositories:

it's not correct, because you can add it on any position of `repositories` closure